hdr_schemata/models/GWDM/latest/schema.json

```diff
@@ -259,6 +281,97 @@
+                                          "type": "null"
+                                    }
+                              ],
+                              "default": null,
+                              "description": "Mental health, Cognitive function",
+                              "title": "Psychological"
+                        },
+                        "physical": {
+                              "anyOf": [
+                                    {
+                                          "$ref": "#/$defs/Physical"
+                                    },
+                                    {
+                                          "type": "null"
+                                    }
+                              ],
+                              "default": null,
+                              "description": "Cardiovascular, Respiratory, Musculoskeletal, Hearing and Vision, Reproductive",
+                              "title": "Physical"
+                        },
+                        "anthropometric": {
+                              "anyOf": [
+                                    {
+                                          "$ref": "#/$defs/Anthropometric"
+                                    },
+                                    {
+                                          "type": "null"
+                                    }
+                              ],
+                              "default": null,
+                              "description": "Height, Weight, Waist circumference, Hip circumference, Blood pressure",
+                              "title": "Anthropometric"
+                        },
+                        "lifestyle": {
+                              "anyOf": [
+                                    {
+                                          "$ref": "#/$defs/Lifestyles"
+                                    },
+                                    {
+                                          "type": "null"
+                                    }
+                              ],
+                              "default": null,
+                              "description": "Cohort lifestyle habits: Smoking, Physical activity, Dietary habits, Alcohol",
+                              "title": "Lifestyle"
+                        },
+                        "socioeconomic": {
+                              "anyOf": [
+                                    {
+                                          "$ref": "#/$defs/SocioEconomic"
+                                    },
+                                    {
+                                          "type": "null"
+                                    }
+                              ],
+                              "default": null,
+                              "description": "Occupation, Family circumstances, Housing, Education, Ethnic group, Martial status, Social support",
+                              "title": "Socio-economic"
```

Comment:
adds fields suggested in https://hdruk.atlassian.net/wiki/spaces/HDT/pages/2635464705/Cohorts+study+requirements+doc